apps/infrastructure/src/iam_policy.tf

```
@@ -32,10 +32,13 @@ data "aws_iam_policy_document" "deploy_github" {
 data "aws_iam_policy_document" "cms_iam_policy" {
```

If you agree, I would create a data for each s3 bucket that limits the cloudfront access to the tenant's medialibrary

```suggestion
data "aws_iam_policy_document" "cms_iam_policy" {
    for_each = {
    for key, config in var.websites_configs :
    key => config
  } 
```